pkg/volume/fc/disk_manager.go

```diff
@@ -36,12 +37,14 @@ type diskManager interface {
 // utility to mount a disk based filesystem
 func diskSetUp(manager diskManager, b fcDiskMounter, volPath string, mounter mount.Interface, fsGroup *int64) error {
 	globalPDPath := manager.MakeGlobalPDName(*b.fcDisk)
-	// TODO: handle failed mounts here.
 	noMnt, err := mounter.IsLikelyNotMountPoint(volPath)
-
-	if err != nil && !os.IsNotExist(err) {
-		glog.Errorf("cannot validate mountpoint: %s", volPath)
-		return err
+	if err != nil {
+		if !os.IsNotExist(err) {
```

Comment:
Please PTAL to #50502  to see what I'm talking about.
These changes should be handled separatly, @saad-ali to you for confirmation